travis/install_avr32-gcc.sh

```diff
@@ -0,0 +1,20 @@
+mkdir avr32-gcc
+cd avr32-gcc
+
+# Get archives
+wget https://gist.github.com/jlecoeur/d83107dda77c4bd62a66/raw/1dc9c3efe832db7cfcd86694693f620d811e9802/avr32-gnu-toolchain-3.4.2.435-linux.any.x86.tar.gz
+wget https://gist.github.com/jlecoeur/d83107dda77c4bd62a66/raw/471a24180afc9c799848e017aa2f8cf10c55cb1a/atmel-headers-6.1.3.1475.zip
```

Comment:
We can either:
- make lis-epfl/MAVRIC public and download from there
- create a new repo in lis-epfl, but IMHO this does not warrant a full repo, and this would be duplicated with MAVRIC/Utilities/Atmel*
- create a gist in my account while MAVRIC is private

I thought the 3d option was the best for now, let me know what you think